driver.sh

```diff
@@ -132,7 +132,7 @@ check_dependencies() {
   command -v ${qemu}
   command -v timeout
   command -v unbuffer
-  command -v clang-8
+  command -v clang-9
   command -v "${LD:="${CROSS_COMPILE:-}"ld}"
```

Comment:
is ld duplicated in this function?

would you mind adding a check for lld-9 while you're here?

Reply:
>is ld duplicated in this function?

Oops, looks like it is... Do you want the duplicate one removed now or later?

>would you mind adding a check for lld-9 while you're here?

But `ld.lld` isn't strictly required to run this script. When we pass `LD=ld.lld-9` to the script on Travis, it will get verified with the `command -v "${LD:="${CROSS_COMPILE:-}"ld}"`

Reply:
good point!  Can be removed in this PR. ☃️

Reply:
Done!